### PR TITLE
Create a 2.0 version that works with Nuxeo 8.10 and Elastic 2.3.5

### PR DESCRIPTION
**Pull request by covolution**

### CODE REVIEW

**Comment by bdelbosc** [comment]
just missing the README that describes expected Nuxeo/ES version and state that it should not report any diff after a first reindex all.

**Comment by covolution** [comment]
The README is on the other branch which will come after this.

**Comment by covolution** [comment]
Added the README updates

**Comment by bdelbosc** [comment]
:+1:

File: src/main/java/org/nuxeo/tools/esync/es/EsDefault.java [hunk 1]
```diff
@@ -264,26 +266,32 @@ public long getOrphanCardinality() {
 
     private Map<String, Long> getTypeCardinalityTimed() {
         LinkedHashMap<String, Long> ret = new LinkedHashMap<>();
+
+        //This doesn't work
```
Comment (efge) [inline src/main/java/org/nuxeo/tools/esync/es/EsDefault.java:270]: ?

Comment (covolution) [reply src/main/java/org/nuxeo/tools/esync/es/EsDefault.java:270]: Sorry about that!  The comment has already been removed on master.

**Pull request merged**